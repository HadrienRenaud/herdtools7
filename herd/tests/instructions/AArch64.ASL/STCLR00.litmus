AArch64 STCLR00
{
uint64_t x=15;
uint64_t 0:X0=5;
0:X1=x;
}
   P0          ;
 STCLR X0,[X1] ;

forall [x]=10
