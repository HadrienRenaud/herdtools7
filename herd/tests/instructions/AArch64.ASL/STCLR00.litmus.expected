Test STCLR00 Required
States 1
[x]=10;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([x]=10)
Observation STCLR00 Always 1 0
Hash=6dce510cf1ec8087064858128b894317

